kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -57,45 +48,98 @@
           >
             {{ getTagline }}
           </KGridItem>
-
+        </KGrid>
+        <div class="tabs">
+          <KButton
+            ref="tab_button"
+            text="Folders"
+            appearance="flat-button"
+            class="tab-button"
+            :appearanceOverrides="customTabButtonOverrides"
+            @click="toggleSidebarView('folder')"
+          />
+          <KButton
+            ref="tab_button"
+            text="Search"
```

Comment:
You have the core string 🙂 

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/mixins/commonCoreStrings.js#L308